vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml

```diff
@@ -0,0 +1,426 @@
+---
+  - name: Define number of BMH's
+    set_fact:
+      NUMBER_OF_BMH: "{{ NUM_OF_MASTER_REPLICAS|int +  NUM_OF_WORKER_REPLICAS|int }}"
+
+  - name: Update maxSurge and maxUnavailable fields
+    shell: |
+           kubectl get machinedeployment -n {{NAMESPACE}} test1 -o json | 
+           jq '.spec.strategy.rollingUpdate.maxSurge=1|.spec.strategy.rollingUpdate.maxUnavailable=1' | 
+           kubectl apply -f-
+    environment:
```

Comment:
I might be wrong, but do we need them as you are already running tests after pivoting?

Reply:
These properties are not affected by pivoting. They properties of machinedeployment.

Reply:
I was referring to:
`environment:
      KUBECONFIG: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"`

Reply:
Yes @furkatgofurov7 because the ansible script is run from source cluster so post pivot all the operations directed towards the pivoted cluster should have this.